charts/argocd-operator/templates/ConsoleLink.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: console.openshift.io/v1
```

Comment:
most of the other templates have a flag to enable if a certain resource is generated; probably good idea to add it here as well. am thinking of a situation where you want to use this on a non-ocp env, I'd presume vanilla k8s will generate an error as it would not understand this kind/apiversion.